### ISSUE TITLE
[FR] Support tasks format

### ISSUE THREAD

**Issue by ltroj** [cfe84/obsidian-pw#5]
This is a really cool plugin and I could see myself using this for my task planning if it would support the popular [obsidian-tasks](https://github.com/obsidian-tasks-group/obsidian-tasks) formatting for tasks e.g. `- [ ] My new task ⏫ 🛫 2024-12-29 ⏳ 2024-12-31 📅 2025-01-04` (that is priority, start date, scheduled date, due date).

Maybe something for future versions!

**Comment by Beingpax** [comment]
Absolutely. This is such a gem. And has such a huge potential. This plugin instantly clicked with me. Supporting the tasks plugin formats would make it so much better. Please look into it

**Comment by F1orian** [comment]
I totally agree! Support for the Tasks Plugin would be great!

**Comment by pcause** [comment]
bump

**Comment by hermanmak** [comment]
Bumping this one, it would synergize well other plugins as a common language. For example, tickticksync. Right now tickticksync will append the Task emoji format as the sync.

**Comment by rkm82** [comment]
+1 from me - given how well established the task plugin is, the "emoji" view should 100% be supported

**Comment by KiraGarvinLerman** [comment]
+1 -- definitely better than my data view set up but it's placed all my prior completed tasks in the backlog instead of recognizing/interacting with the metadata.

**Comment by texastoland** [comment]
It looks as easy as making this regex more flexible:

https://github.com/cfe84/obsidian-pw/blob/bb5597937698a0901f70fc1e1deaccc65e402f1b/src/domain/LineOperations.ts#L32

I could do that but the author may not be interested (based on silence).

**Comment by Volker-brdb** [comment]
if this feature is realized i will instantly start using this really great plugin. without this feature it's unfortunatley useless for me

**Comment by Evthestrike** [comment]
+1

I have been looking for a plugin like this!

**Comment by Ravenclawer** [comment]
VOTE FOR THIS FEATURE

**Comment by NTTAKR** [comment]
Any plan if/when you will be doing this?

It's the one thing currently preventing me from using your great addon, because I would need to reformat all my tasks.

**Comment by sarahwarchhold** [comment]
yes, that would really make things easier, thanks for your work! :-)

**Comment by bane73** [comment]
I just want to echo what others have said here! My current attempt to implement similar to this plugin is to create a bunch of separate nested queries that live in callouts and columns... and it seems it's all gotten too heavy for my individual daily notes and severe lag kicks in after about 5-8hrs of constant use.

THIS looks very promising! But it def needs to support the standard Task plugin date syntax.

 Including options with how I want it to treat Start vs Scheduled vs Due. Would also be excellent if I can define a box or three to focus on a certain tag, as opposed to a date -- for example a box of all tasks that have already "Started" and are tagged with "#follow-up" -- this allows me to create a future-task which does not need to be done on a specific date but is a reminder for me to follow-up with someone/something but doesn't enter my view until a certain start-date. Finally, any tasks that are unscheduled (no start/scheduled) or overdue, should go into a box "Mis-scheduled". 

Could you allow us to create boxes with our own Task queries?
ie:
```tasks
tags do not include #ignore
(due before <today>) OR (no start date) OR (scheduled before <today>)
not done
sort by urgency
sort by due date
sort by start date
show urgency
```

If you could do that, robustly, I would seriously pay for it LOL!
